java/amazon-kinesis-producer/src/main/java/com/amazonaws/services/kinesis/producer/KinesisProducerConfiguration.java

```diff
@@ -137,7 +142,76 @@ public KinesisProducerConfiguration setMetricsCredentialsProvider(AWSCredentials
         this.metricsCredentialsProvider = metricsCredentialsProvider;
         return this;
     }
-    
+
+    /**
+     * Return the set {@link GlueSchemaRegistryConfiguration} that has
+     * configuration options for Glue Schema Registry serializer to function.
+     * @return GlueSchemaRegistryConfiguration
+     */
+    public GlueSchemaRegistryConfiguration getGlueSchemaRegistryConfiguration() {
+        return glueSchemaRegistryConfiguration;
+    }
+
+    /**
+     * Set the {@link GlueSchemaRegistryConfiguration} that has configuration options for
+     * Glue Schema Registry serializer to function. If this is set explicitly, it will be used.
+     * If not, a default will be created at runtime.
```

Comment:
In the java docs for these new configs in KPL configuration can we clarify that this is not a mandatory feature but rather additionally available for use. Stating here that a default will be created is confusing since customers might think this feature is always on.